ethereum/blockcreation/src/main/java/org/hyperledger/besu/ethereum/blockcreation/AbstractBlockCreator.java

```diff
@@ -214,9 +214,7 @@ protected BlockCreationResult createBlock(
       final BlockHeader blockHeader = createFinalBlockHeader(sealableBlockHeader);
 
       final Block block =
-          new Block(
-              blockHeader,
-              new BlockBody(transactionResults.getTransactions(), ommers, Optional.empty()));
+          new Block(blockHeader, new BlockBody(transactionResults.getTransactions(), ommers));
```

Comment:
Isn't this the one we need withdrawals in - or will that be wired up later?

Reply:
Yeah, we definitely will need to do that but wanted to just keep this PR about the existing state which doesn't pass in the withdrawals. I will add in the next PR that includes all the wiring changes.